packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -629,28 +661,15 @@ class DDSCarousel extends HostListenerMixin(StableSelectorMixin(LitElement)) {
     }
   }
 
-  markHiddenAsInert() {
```

Comment:
This turned out to be redundant now that the intersection observer works more reliably.